sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1695,7 +1698,7 @@ public void validate(PCollection<TableRow> input) {
             "CreateDisposition is CREATE_IF_NEEDED, however no schema was provided.");
 
         // The user specified a table.
-        if (jsonTableRef != null && validate) {
+        if (jsonTableRef != null) {
```

Comment:
Target use case is running something against a pre-existing table (or tables) so that I don't have to provide a schema -- maybe there's a better way?